diagnostic.md

```diff
@@ -36,13 +41,18 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/$ID
```

Comment:
Again, this only works if you actually set that variable. This would work:

```sh
curl --include --request DELETE http://www.example.com/donuts/1
```